js/templates/modals/settings/moderation.html

```diff
@@ -5,12 +5,15 @@ <h2 class="h3 clrT">Moderation</h2>
       <a class="btn clrP clrBr clrSh2 modalContentCornerBtn js-save"><%= ob.polyT('settings.btnSave') %></a>
```

Comment:
There's a layout issue:

1. set the fee type to percentage.
2. click save.
3. set the fee type to fixed.

The currency field is cut-off:

![image](https://cloud.githubusercontent.com/assets/794517/22379965/706ae746-e470-11e6-96d8-3086e205b5b9.png)